### DISCUSSION TITLE
I would like to ask you that if we can swipe up in the app.

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#133] [Q&A] [answered]
I am trying on a project . The idea is that if we swipe down we could put something like text. For example if we swipe down at home screen it opens up the control center. But in app I wanted to put the text as we launch the control centre. So can you help me . 

The code is not working for me

```javascript
const isVertical = true
      hmUI.setScrollView(true, -334, 1, isVertical);
hmUI.createWidget(hmUI.widget.FILL_RECT, {
      x: 20,
      y: -334,
      w: 100,
      h: 200,
      radius: 20,
      color: 0xFFAF60
    })

``` 
Please tell me my errors or what should I do it to make the code work

**Comment by MyoungXUE** [comment, marked as answer]
`hmUI.setScrollView` cannot be set to a negative value. This API only changes the up and down scrolling behavior of the page without affecting the drawing and layout of the control.

You can use `hmUI.scrollToPage` when entering the page to directly turn to the second page, so that when entering the page, slide down to display the first page view. To simulate a similar effect.

**Reply by alexmathew11** [reply]
But it would be great if we can put scroll,s view to the negative value
